src/lib.rs

```diff
@@ -11,7 +11,7 @@
 //! let github = Github::new(
 //!   String::from("user-agent-name"),
 //!   Credentials::Token(
-//!     String::from("personal-access-token")
+//!     String::from("personal-access-token".to_string())
```

Comment:
`String::from("personal-access-token")` is absolutely fine here

Reply:
Good catch @nickelc.
:disappointed: I wasn't even able to provide a PR which changes 12 characters in doc ...

Reply:
#267 

CC: @softprops